src/main/java/com/ns/task/common/Product.java

```diff
@@ -0,0 +1,40 @@
+package com.ns.task.common;
+
+public class Product {
+  private Integer id;
+  private String name;
+  private String description;
+  private Double price;
```

Comment:
Is it the best practice to use Double for money management?. 

Please run these lines in a test and tell me what's wrong in it. 
```
double a = 2.00;
double b = 1.10;
System.out.println(a-b);
```